core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -207,6 +206,10 @@ public static Logger getLogger() {
         return logger;
     }
 
+    public String getConfigFileName() {
```

Comment:
Dunno if I wanna expose it only to be visible in the tests. Maybe you should use [@VisibleForTesting](https://developer.android.com/reference/android/support/annotation/VisibleForTesting.html)

Reply:
Didn't know that existed, it looks just like what I wanted

Reply:
That is why I'm here! :bowtie: